vignettes/how-to-bench.Rmd

```diff
@@ -0,0 +1,225 @@
+---
```

Comment:
Consider running the follow to solve style inconsistencies.

```r
styler::style_file(
    path = usethis::proj_path("vignettes", "how-to-bench.Rmd"),
    transformers = biocthis::bioc_style()
)
```

In the other hand, consider to break down your explanation lines to lines of 80 width, this would not affect the outcome text.